packages/template-retail-react-app/app/hooks/use-password-reset.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2024, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import {AuthHelpers, useAuthHelper} from '@salesforce/commerce-sdk-react'
+import {useToast} from '@salesforce/retail-react-app/app/components/shared/ui'
+import {useIntl} from 'react-intl'
+import {absoluteUrl} from '@salesforce/retail-react-app/app/utils/url'
+/**
+ * This hook provides commerce-react-sdk hooks to simplify the reset password flow.
+ */
+export const usePasswordReset = () => {
```

Comment:
Alternatively, we can separate  `getPasswordResetToken` and `resetPassword` as different hooks. Is there a good reason to keep them together?

Reply:
The reason we keep them together is because they are the 2 apis used for the password reset flow. So keeping them together in the same file makes logical sense

Reply:
I also like having the 2 apis in the same hook. I removed the default export however, so there is only 1 way to import the hook now, with the curly braces.

`import {usePasswordReset} from '@salesforce/retail-react-app/app/hooks/use-password-reset'`